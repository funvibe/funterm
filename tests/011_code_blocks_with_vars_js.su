js (greeting, message) {
    function greeting(hello) {
        console.log(hello)
    }
    message = "hello again"
}
js.greeting("hello from js")
js.print(js.message)