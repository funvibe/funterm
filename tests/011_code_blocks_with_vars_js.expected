hello from js
hello again